samples/filter_sort_groups.py

```diff
@@ -0,0 +1,86 @@
+####
+# This script demonstrates how to filter groups using the Tableau
+# Server Client.
+#
+# To run the script, you must have installed Python 2.7.9 or later.
+####
+
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Filter on groups')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+    parser.add_argument('-p', default=None)
+    args = parser.parse_args()
+
+    password = ''
+    if args.p is None:
+        password = getpass.getpass("Password: ")
+    else:
+        password = args.p
+
+    # Set logging level based on user input, or error by default
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
+
+    tableau_auth = TSC.TableauAuth(args.username, password)
+    server = TSC.Server(args.server)
+
+    with server.auth.sign_in(tableau_auth):
+        server.version = '2.7'
+        group_name = 'SALES NORTHWEST'
+        # Try to create a group named "SALES NORTHWEST"
+        try:
+            group1 = TSC.GroupItem(group_name)
+            group1 = server.groups.create(group1)
+            print(group1)
```

Comment:
Do we want to print something more meaningful than the repr of a group?

eg "Group created: <repr>"